packages/unified-selection/src/test/_helpers/SelectableObjectCreator.ts

```diff
@@ -0,0 +1,35 @@
+import faker from "@faker-js/faker";
```

Comment:
I don't think it's worth having random data for tests. It also adds additional dependency. Can you remove `faker` dependency?